.github/workflows/ci-github-actions-self-hosted.yaml

```diff
@@ -115,13 +115,13 @@ jobs:
       fail-fast: false
       matrix:
         jobname: [
-            Clang15-MPI-CUDA-AFQMC-Offload-Real-Mixed, # auxiliary field, offload
-            Clang15-MPI-CUDA-AFQMC-Offload-Real,
-            Clang15-MPI-CUDA-AFQMC-Offload-Complex-Mixed,
-            Clang15-MPI-CUDA-AFQMC-Offload-Complex,
-            Intel21-MPI-CUDA-AFQMC-Real-Mixed, # auxiliary field, requires MPI
-            Intel21-MPI-CUDA-AFQMC-Complex-Mixed,
-            Intel21-MPI-CUDA-AFQMC-Real,
+            TeslaV100-Clang16-MPI-CUDA-AFQMC-Offload-Real-Mixed, # auxiliary field, offload
```

Comment:
TeslaV100->NV_V100.

Reply:
Yes, Tesla is redundant here. Can we use hyphens - everywhere or do you have a reason for underscores?

Reply:
V100 should be fine on its own if we are happy with RadeonVII

Reply:
Just V100? CUDA is in the name.

Reply:
Just V100 is fine. I also use R7 instead of RadeonVII.